common/k8s/src/kube_stats/helpers.rs

```diff
@@ -0,0 +1,191 @@
+pub fn convert_cpu_usage_to_milli(cpu: &str) -> i32 {
```

Comment:
returns some number in all cases, no errors. zero could be misleading value.

Reply:
really good call